website/app/controllers/show.js

```diff
@@ -20,16 +24,21 @@ export default class ShowController extends Controller {
   }
 
   get renderedContent() {
+    // schedule tabs logic for after this content is rendered
+    if (!this.fastboot.isFastBoot) {
+      schedule('afterRender', () => {
+        this.didInsertContent();
+      });
+    }
     const converter = new showdown.Converter(showdownConfig);
     return converter.makeHtml(this.model.content);
   }
 
-  @action
-  didInsertContent(docPageContentElement) {
+  didInsertContent = () => {
     let sections = [];
     let tabs = [];
     let tocs = [];
-    docPageContentElement
+    document
       .querySelectorAll(`section[id^=section-]`)
```

Comment:
since we changed the parent that we use to execute the `querySelectorAll` DOM query, I would prefer to be more specific then in the selection

```suggestion
      .querySelectorAll(`.doc-page-content section[id^=section-]`)
```